data/successors/successor outfits.txt

```diff
@@ -538,38 +538,37 @@ outfit `"Veusa" Inductive Reverser`
 	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
 
 
-outfit "Multimodal Armor: Heavy"
+outfit "Multimodal Armor: Light"
 	category "Systems"
 	series "Special Systems"
 	index 11010
-	thumbnail "outfit/successor armor heavy"
+	thumbnail "outfit/successor armor light"
 	licenses
 		"High Houses"
-	"hull protection" .20
+	"hull protection" 0.10
+	"inertia reduction" 0.15
 	"multimodal armor" -1
 	"unplunderable" 1
 	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
-	description "	This configuration strengthens the bonds between the atoms making up the armor, increasing its resistance to direct damage."
+	description "	This configuration introduces well-ordered nanoscale voids into the armor's structure, reducing a warship's mass without compromising structural integrity."
 
-outfit "Multimodal Armor: Keystone"
+outfit "Multimodal Armor: Heavy"
 	category "Systems"
 	series "Special Systems"
-	index 11040
-	thumbnail "outfit/successor armor keystone"
+	index 11020
+	thumbnail "outfit/successor armor heavy"
 	licenses
 		"High Houses"
-	cost 10000000
-	"quantum keystone" 1
-	"scan interference" 3
+	"hull protection" .20
```

Comment:
This should probably be standardized across the different Successor outfits at least, either by switching this to
```suggestion
	"hull protection" 0.2
```

(or I guess 0.20) or by removing the leading zeroes on others.